test/5.future-and.test.js

```diff
@@ -2,113 +2,65 @@
 
 const expect = require('chai').expect;
 const Future = require('../fluture.js');
-const FutureAnd = Future.classes.FutureAnd;
```

Comment:
I prefer this new way of testing, because it tests behavior rather than implementation.